js/main.js

```diff
@@ -25,4 +26,100 @@ $form.addEventListener('submit', function (event) {
   $img.setAttribute('src', './images/placeholder-image-square.jpg');
   $form.reset();
 
+  $entries.className = 'view entries container';
+  $entryForm.className = 'view entry-form hidden container';
+  data.view = 'entries';
+
+  var $ul = document.querySelector('ul');
+  var $newEntry = createList(newObj);
+  $ul.prepend($newEntry);
+  $noEntries.className = 'hidden';
 });
+
+function createList(data) {
+  var $li = document.createElement('li');
+  $li.setAttribute('class', 'row');
+
+  var $imgElement = document.createElement('img');
+  $imgElement.setAttribute('class', 'column-half');
+  $imgElement.setAttribute('src', data.photo);
+
+  $li.appendChild($imgElement);
+
+  var $div = document.createElement('div');
+  $div.setAttribute('class', 'column-half');
+
+  $li.appendChild($div);
+
+  var $h1 = document.createElement('h1');
+  $h1.textContent = data.title;
+
+  $div.appendChild($h1);
+
+  var $p = document.createElement('p');
+  $p.textContent = data.notes;
+
+  $div.appendChild($p);
+
+  return $li;
+
+}
+
+window.addEventListener('DOMContentLoaded', function (event) {
+  var $ul = document.createElement('ul');
+
+  for (var i = 0; i < data.entries.length; i++) {
+    var result = createList(data.entries[i]);
+    $ul.appendChild(result);
+  }
+});
+
+var $buttonNew = document.querySelector('.button-new');
+var $entryTab = document.querySelector('.entry-tab');
+var $entries = document.querySelector('.entries');
+var $entryForm = document.querySelector('.entry-form');
+var $formTab = document.querySelector('.form-tab');
+
+$buttonNew.addEventListener('click', function (event) {
+  if ($entryForm !== '') {
```

Comment:
`$entryForm` is a DOM element, it will never be strictly equal to an empty string. This conditional can be removed.